packages/network-visibility-controller/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@metamask/network-visibility-controller",
+  "version": "0.0.0",
+  "description": "Handles user-facing network state for how networks are displayed and interacted with",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/network-visibility-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.cjs",
+  "types": "./dist/index.d.cts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build": "ts-bridge --project tsconfig.build.json --verbose --clean --no-references",
+    "build:docs": "typedoc",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/network-visibility-controller",
+    "since-latest-release": "../../scripts/since-latest-release.sh",
+    "changelog:update": "../../scripts/update-changelog.sh @metamask/network-visibility-controller",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest --reporters=jest-silent-reporter",
+    "test:clean": "NODE_OPTIONS=--experimental-vm-modules jest --clearCache",
+    "test:verbose": "NODE_OPTIONS=--experimental-vm-modules jest --verbose",
+    "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch"
+  },
+  "devDependencies": {
+    "@lavamoat/allow-scripts": "^3.0.4",
+    "@metamask/auto-changelog": "^3.4.4",
+    "@metamask/error-reporting-service": "^1.0.0",
+    "@metamask/network-controller": "^23.6.0",
+    "@metamask/providers": "^22.1.0",
+    "@types/jest": "^27.4.1",
+    "deepmerge": "^4.2.2",
+    "jest": "^27.5.1",
+    "ts-jest": "^27.1.4",
+    "typedoc": "^0.24.8",
+    "typedoc-plugin-missing-exports": "^2.0.0",
+    "typescript": "~5.2.2"
+  },
+  "engines": {
+    "node": "^18.18 || >=20"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "dependencies": {
+    "@metamask/base-controller": "^8.0.1",
+    "@metamask/keyring-api": "^18.0.0",
+    "@metamask/keyring-snap-client": "^5.0.0",
```

Comment:
Do we use this dep, I don't see it when I search for it?

Reply:
We are inheriting the `BaseController` in `NetworkVisibilityController` so that we can extend from it. I saw this pattern elsewhere in the app. Did I miss something here?